model-optimizer/extensions/front/mxnet/Swish_ext.py

```diff
@@ -14,24 +14,28 @@
  limitations under the License.
 """
 
-from extensions.ops.activation_ops import Sigmoid
-from extensions.ops.elementwise import Mul
+import numpy as np
```

Comment:
Why this file is called "Swish_ext.py" while it is a transformation? It should be an extractor, right?